src/lib/datepicker/calendar.ts

```diff
@@ -244,14 +258,15 @@ export class MdCalendar<D> implements AfterContentInit {
       case ENTER:
         if (this._dateFilterForViews(this._activeDate)) {
           this._dateSelected(this._activeDate);
-          break;
+          event.preventDefault();
```

Comment:
There wasn't anything I ran into, it was more of just a precaution, should I remove it?